### PR TITLE
Bump NodeJS to 16

### PR DESCRIPTION
**Pull request by theomelo**
Fixes https://github.com/exercism/pr-commenter-action/issues/41

This PR upgrades Node 16 and checkout to v3

### CODE REVIEW

**Comment by angelikatyborska** [comment]
Thank you! I wouldn't have noticed the node 12 problem myself 🙁. But I think those changes are not enough - what you're changing is the node version used for testing the action. We also need to change the node version used for running the action when people use it. That's defined in `action.yml`. Could you change that too?

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -9,11 +9,10 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@dcd71f646680f2efd8db4afa5ad64fdcba30e748
-      - name: Setup NodeJS
+      - uses: actions/checkout@3
```
Comment (angelikatyborska) [inline .github/workflows/test.yml:12]: Could you undo those changes? Checkout is already at version 3, that's what the commit sha points to (https://github.com/actions/checkout/commit/dcd71f646680f2efd8db4afa5ad64fdcba30e748). Dependabot takes care of updating it in this repository. I also don't see a reason to remove the step name 🤔

Comment (theomelo) [reply .github/workflows/test.yml:12]: That was an oversight. My apologies for that 🙇 I am fixing it in my branch, and I'll update the PR in a bit. 🙏

**Comment by theomelo** [comment]
> Thank you! I wouldn't have noticed the node 12 problem myself 🙁. But I think those changes are not enough - what you're changing is the node version used for testing the action. We also need to change the node version used for running the action when people use it. That's defined in `action.yml`. Could you change that too?

It should be fixed now 🙌

**Comment by angelikatyborska** [comment]
Thank you! I forgot that `.tool-verisons` also needed updating, but I did that myself now 🙂

**Review by angelikatyborska — APPROVED** [review]

**Pull request merged**